### PR TITLE
Add missing UPGRADE notes for 7.1

### PR DESCRIPTION
**Pull request by wouterj**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

### CODE REVIEW

File: UPGRADE-7.1.md [hunk 1]
```diff
@@ -69,6 +120,7 @@ Form
 FrameworkBundle
 ---------------
 
+ * [BC BREAK] Enabling `framework.rate_limiter` requires `symfony/rate-limiter` 7.1 or higher
```
Comment (stof) [inline UPGRADE-7.1.md:123]: Does FrameworkBundle 7.1 conflict with rate-limiter < 7.1 ? If yes, there is no BC break as composer deals with ensuring compatible versions.

Comment (wouterj) [reply UPGRADE-7.1.md:123]: No, it doesn't. It's only part of the `require-dev` of framework bundle.

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @wouterj.

File: UPGRADE-7.1.md [hunk 2]
```diff
@@ -50,17 +52,66 @@ DependencyInjection
...
+       ) {
+       }
+   }
+   ```
+
 DoctrineBridge
 --------------
 
- * Deprecated `DoctrineExtractor::getTypes()`, use `DoctrineExtractor::getType()` instead
+ * Deprecate `DoctrineExtractor::getTypes()`, use `DoctrineExtractor::getType()` instead
```
Comment (chalasr) [inline UPGRADE-7.1.md:94]: This one should be removed actually https://github.com/symfony/symfony/commit/eb319a2cbfdcecb290f2023cee2ad524b305e20b

Comment (xabbuh) [reply UPGRADE-7.1.md:94]: fixed in ecd1d92161

**Pull request merged**